tests/langchain/test_langchain_model_export.py

```diff
@@ -197,6 +219,36 @@ def test_langchain_log_huggingface_hub_model_metadata(model_path):
     assert loaded_model.prompt.template == "What is a good name for a company that makes {product}?"
 
 
+def test_langchain_agent_model_predict():
```

Comment:
Can we test serving with `mlflow models serve` as well? That's super important to ensure that the model can be loaded in an environment where the tools aren't already defined in the current REPL.